docs/setuptools.rst

```diff
@@ -114,13 +127,16 @@ assume your directory structure changed to this::
         scripts/
             __init__.py
             yourscript.py
+    setup.py
```

Comment:
Add a top-level `project/` folder.

Reply:
On top of the entire hierarchy? Doesn't that then make the listing too verbose?